### ISSUE TITLE
Send Text and Pictures in one Message

### ISSUE THREAD

**Issue by bokf** [deltachat/deltachat-android#54]
Would be nice to write some text together with a picture. Most of my contacts receive the message as email and now get text and pictures as separate messages. When I send a picture I can send it, but not write any message before sending.

**Comment by r10s** [comment]
Maybe we will add this later on, for the moment, we just want to Keep-It-Simple

**Comment by andreygursky** [comment]
In case someone will start to implement this: *please* do not force every user to download huge "messages". Instead consider **configurable** auto-download of such embedded images (attachments?). It is not that one always have a good connection and/or enough high-speed traffic. By having this in mind before starting with implementation, it should be much easier to handle this before it turns into an issue which will be harder to fix afterwards.

**Comment by yt42** [comment]
I fully agree of what @andreygursky say.

**Comment by bokf** [comment]
Combining a text message with a picture message has nothing to do with the picture size. This is another topic.

**Comment by r10s** [comment]
I think, for now, this is out of scope of Delta Chat unless there are some additional volunteers to implement the feature. However, the request stay available eg. via https://github.com/deltachat/deltachat-android/wiki

**Comment by r10s** [comment]
this will be possible in the upcoming android-dev :)

**Comment by testbird** [comment]
> Combining a text message with a picture message has nothing to do with the picture size. This is another topic.

Sure, the text and picture size are unrelated, but when things are sent in one large message, it is necessary to download all attachments, only to be able to fetch the text.

A short attachment description like a short title or footnote (e.g. a picture description to be shown with the picture in the gallery) could be sent in an unencrypted subject (or in an encrypted email header), and may be read without requiring to download all attachments.
However, with the current protocol currently only sending attachments in separate emails will allow a traffic saving mode that downloads attachments only on request.
https://github.com/deltachat/deltachat-core/issues/436
https://github.com/deltachat/deltachat-core/issues/437

Is this really a frontend issue?
